manager/src/main/java/com/redhat/developer/manager/dao/ConnectorDAO.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.developer.manager.dao;
+
+import java.util.List;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.transaction.Transactional;
+
+import com.redhat.developer.manager.models.Connector;
+import com.redhat.developer.manager.models.ConnectorStatus;
+
+import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;
+import io.quarkus.panache.common.Parameters;
+
+@ApplicationScoped
+@Transactional
+public class ConnectorDAO implements PanacheRepositoryBase<Connector, String> {
+
+    public List<Connector> findByStatus(ConnectorStatus status) {
+        Parameters params = Parameters
+                .with("status", status);
+        return find("#CONNECTOR.findByStatus", params).list();
+    }
+
+    public List<Connector> findByNameAndCustomerId(String name, String customerId) {
```

Comment:
I think this should return a `Connector`. We should add a unique constraint on the DB level that we can't store two Connectors with the same `customerId` and `name` so this query either returns `null` or the `Connector`.

Reply:
The unique constraint is already present at schema level